### ISSUE TITLE
datetime package was obsoleted by datetime2

### ISSUE THREAD

**Issue by cebamps** [Hightor/gitinfo2#15]
The package [datetime.sty](http://ctan.org/pkg/datetime) was recently obsoleted by [datetime2.sty](http://ctan.org/pkg/datetime2). gitinfo2 is not compatible with the new version.

This worked for me:

``` LaTeX
\@ifpackageloaded{datetime2}{%
    \IfEq{\gitAuthorDate}{\gitInf@missing}{%
        \DTMsavedate{gitdate}{\THEYEAR-\THEMONTH-\THEDAY}
    }{%
        \StrCut{\gitAuthorDate}{-}{\gitInf@year}{\gitInf@md}
        \StrCut{\gitInf@md}{-}{\gitInf@month}{\gitInf@day}
        \DTMsavedate{gitdate}{\gitInf@year-\gitInf@month-\gitInf@day}
    }    
}{}%
```

**Comment by longborough** [comment]
Thank you for pointing this out. A while back, I agreed with Dr. Nicola Talbot (the author of datetime{,2}, to leave this alone, for now, while datetime2 is "settling down". Clearly, it's now time to revisit that, and I hope to do this sometime in the next two months. I'll leave this issue open for now.

**Comment by longborough** [comment]
Fixed by  commit  28766d0, adapted from you code. Thanks!

For release 2.0.5
